packages/radix-ui-themes/src/components/alert-dialog.css

```diff
@@ -2,4 +2,5 @@
 }
 
 .rt-AlertDialogContent {
+  position: relative;
 }
```

Comment:
We don't need this style as Alert Dialog re-uses `rt-DialogContent` class

Should we delete the `alert-dialog.css` file altogether? Safe to say the design won't diverge from the base dialog

Reply:
I can follow up with a cleanup PR 👍🏻